asserts/store_asserts.go

```diff
@@ -0,0 +1,61 @@
+package asserts
+
+import "fmt"
+
+// EnterpriseStore holds an enterprise-store assertion, defining the
+// configuration needed to connect a device to the enterprise store.
+type EnterpriseStore struct {
+	assertionBase
+}
+
+// OperatorID returns the account id of the enterprise store's operator.
+func (estore *EnterpriseStore) OperatorID() string {
+	return estore.HeaderString("operator-id")
+}
+
+// Store returns the name of the operator's enterprise store.
```

Comment:
The intention is for the store name to be human readable, where tools and UIs can assume "default" if no explicit store name is provided.